frontend/public/components/storage/attach-storage.tsx

```diff
@@ -387,10 +448,13 @@ export type AttachStorageFormState = {
   mountPath: string;
   subPath: string;
   mountAsReadOnly: boolean;
-  error: any;
+  error: string;
   allContainers: boolean;
-  containers: any;
+  containers: Object;
   volumeAlreadyMounted: boolean;
+  showCreatePvc: string;
+  pvcObj: Object;
```

Comment:
`Object` isn't all that helpful. Is this a Kubernetes PVC object? If so, use `: K8sResourceKind`